### DISCUSSION TITLE
Turn off rendering to speed up the training process

### DISCUSSION THREAD

**Discussion by wjh-scut** [microsoft/AirSim#4804] [Support Q&A]
I'm using MADDPG to train multi-UAV for navigation tasks, but the whole training process takes a lot of time.I would like to ask, can turning off the rendering of UE4 help speed up the training process of RL? If so, how should I turn off rendering?

**Comment by alonfaraj** [comment]
You can use `"ViewMode": "NoDisplay"` in settings.json to turn off the rendering and see if it helps.

**Reply by wjh-scut** [reply]
Ok I'll give it a try, thanks!
